Sources/MapboxMaps/Annotations/Generated/CircleAnnotationManager.swift

```diff
@@ -0,0 +1,249 @@
+                            category: "Annotations")
+            }
+        }
+    }
+        
+    /// Controls the scaling behavior of the circle when the map is pitched.
+    public var circlePitchScale: CirclePitchScale? {
+        didSet {
+            do {
+                try style?.setLayerProperty(for: layerId, property: "circle-pitch-scale", value: circlePitchScale?.rawValue as Any)
+            } catch {
+                Log.warning(forMessage: "Could not set CircleAnnotationManager.circlePitchScale due to error: \(error)",
+                            category: "Annotations")
+            }
+        }
+    }
+        
+    /// The geometry's offset. Values are [x, y] where negatives indicate left and up, respectively.
+    public var circleTranslate: [Double]? {
+        didSet {
+            do {
+                try style?.setLayerProperty(for: layerId, property: "circle-translate", value: circleTranslate as Any)
+            } catch {
+                Log.warning(forMessage: "Could not set CircleAnnotationManager.circleTranslate due to error: \(error)",
+                            category: "Annotations")
+            }
+        }
+    }
+        
+    /// Controls the frame of reference for `circle-translate`.
+    public var circleTranslateAnchor: CircleTranslateAnchor? {
+        didSet {
+            do {
+                try style?.setLayerProperty(for: layerId, property: "circle-translate-anchor", value: circleTranslateAnchor?.rawValue as Any)
+            } catch {
+                Log.warning(forMessage: "Could not set CircleAnnotationManager.circleTranslateAnchor due to error: \(error)",
+                            category: "Annotations")
+            }
+        }
+    }
+    
+    // MARK: - Selection Handling -
+
+    /// Set this delegate in order to be called back if a tap occurs on an annotation being managed by this manager.
+    public weak var delegate: AnnotationInteractionDelegate? {
+        didSet {
+            if delegate != nil {
+                setupTapRecognizer()
+            } else {
+                guard let view = view, let recognizer = tapRecognizer else { return }
+                view.removeGestureRecognizer(recognizer)
+                tapRecognizer = nil
+            }
+        }
+    }
+
+    /// The `UITapGestureRecognizer` that's listening to touch events on the map
+    private var tapRecognizer: UITapGestureRecognizer?
+
+    internal func setupTapRecognizer() {
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap(_:)))
+        tapGestureRecognizer.numberOfTapsRequired = 1
+        tapGestureRecognizer.numberOfTouchesRequired = 1
+        view?.addGestureRecognizer(tapGestureRecognizer)
+        tapRecognizer = tapGestureRecognizer
+    }
+    
+    @objc internal func handleTap(_ tap: UITapGestureRecognizer) {
+        let options = RenderedQueryOptions(layerIds: [layerId], filter: nil)
```

Comment:
We previously talked about this being on the orchestrator, but that seems like an optimization that we can address later.

Reply:
let's talk about this.. Not sure if it's something we want anymore.